recipes/qt/5.x.x/test_package/conanfile.py

```diff
@@ -10,6 +10,7 @@ class TestPackageConan(ConanFile):
     generators = "qt", "cmake", "cmake_find_package_multi"
 
     def build_requirements(self):
+        self.build_requires("cmake/3.20.2")
```

Comment:
For qt6 this was needed because qt itself requires recent cmake. Why is this needed here ?

Reply:
Because in the build() function of this test package we build also with cmake:
```
def build(self):
        self._build_with_qmake()
        self._build_with_meson()
        self._build_with_cmake_find_package_multi()
```

Reply:
yes, but cmake is already pre-required for ~100% of recipes on CCI. If build passes with the cmake version shipped with MSVC, and the versions in https://github.com/conan-io/conan-docker-tools images, then there is no need to be explicit about it in `build_requirements`

Reply:
Ok, will remove that from build_requirements then.